src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -362,6 +374,40 @@ export class NetworkAreaDiagramViewer {
             }
         }
     }
+    private onHover(mouseEvent: MouseEvent) {
+        const equipmentsWithPopover = ['LINE', 'TWO_WINDINGS_TRANSFORMER', 'PHASE_SHIFT_TRANSFORMER'];
```

Comment:
ok, I understand why you did this, following the same logic as in SLD. Indeed, there's no need to proceed in the case of your project using powsybl, even if I'm surprised you dismiss TieLines / DanglingLines. But anyway you're not the only user of this library. In powsybl we try to remain generic so it can be reused by other application with other similar needs.